tests/foreman/cli/test_import.py

```diff
@@ -273,6 +273,62 @@ def test_reimport_orgs_default_negative(self, test_data):
             'rm -rf ${{HOME}}/.transition_data {}'.format(files['users'])
         )
 
+    @data(*import_org_data())
+    def test_import_orgs_recovery(self, test_data):
+        """@test: Try to Import organizations with the same name to invoke
+        usage of a recovery strategy (rename, map, none)
+
+        @feature: Import Organizations Recover
+
+        @assert: 2nd Import will rename the new organizations, 2nd import will
+        map them and the 3rd one will resul in No Action Taken
+
+        """
+        # prepare the data
+        files = dict(self.default_dataset[1])
+        new_dataset = update_csv_values(files['users'], test_data)
+        ssh.upload_file(new_dataset, new_dataset)
+        files['users'] = new_dataset
+        # initial import
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0)
+        # clear the .transition_data to clear the transition mapping
+        ssh.command('rm -rf ${HOME}/.transition_data')
+        # use the default (rename) strategy
+        self.assertIn(
```

Comment:
I am a huge fan of using `return_code` verification myself, but this is not a case where checking the code is sufficient. Even when something goes wrong and nothing is being imported, the `return_code` remains 0, with "No action taken" on `stdout`.
There are also many more scenarios, where `return_code` = 0 yet the result is not right, e.g. 2 of 3 users/orgs being imported, etc.

..what do you think?